### PR TITLE
fix(react-layout): fix componentWillUnmount error

### PR DESCRIPTION
**Pull request by younkue**
## Issue

#138

### CODE REVIEW

File: packages/react-layout/src/Layout.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ export default class Layout extends Component {
 		}
 	}
 	_newItem(element) {
-		const id = (new Date().getTime() + Math.floor(Math.random() * 1000));
```
Comment (jongmoon) [inline packages/react-layout/src/Layout.js]: @younkue `1000` was not enough? (Maybe it depends on browser and device).. Can`newItem` be called several times in 1ms? It's just question

Comment (younkue) [reply packages/react-layout/src/Layout.js]: It's not enough :(

File: packages/react-layout/examples/html/GridLayout.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
 <style>
     .item {
         position: absolute;
-        width: 200px;
+        width: 15%;
```
Comment (jongmoon) [inline packages/react-layout/examples/html/GridLayout.html]: @younkue I wonder why did you change it to percentage.
